.github/workflows/review.yml

```diff
@@ -112,3 +113,33 @@ jobs:
         with:
           msg: "Created review app at https://ecf-review-pr-${{ github.event.number }}.london.cloudapps.digital"
           check_for_duplicate_msg: true
+
+      - name: Trigger Smoke Test
+        uses: benc-uk/workflow-dispatch@v1.1
+        with:
+          workflow: Smoke Test
+          token: ${{ secrets.PERSONAL_TOKEN }}
```

Comment:
sorry, what is this token used for? and can it be named more aptly?

Reply:
To dispatch an action you cannot use a default github token provided in the environment. You need a 'personal' token. 

I can change the secret name to something more apt if you want? 

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token